imaging.py

```diff
@@ -0,0 +1,83 @@
+"""
+Taking images with a USB camera on a computer on the drone.
+"""
+import pathlib
+import time
+
+import cv2
+
+
+CAMERA_NAME = 0
+# TODO: Move this to configuration file
```

Comment:
Should this TODO message be applied to `CAMERA_NAME` as well or only `CAMERA_WIDTH` and `CAMERA_HEIGHT`. If it applies to all 3 then I suggest moving the TODO message above all 3 constants so it more clear.